include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -977,24 +977,22 @@ struct CustomOpApi {
 
   void ThrowOnError(OrtStatus* result);
 
-  void CreateOpAttr(_In_ const char* name,
-                    _In_ const void* data,
-                    _In_ int len,
-                    _In_ OrtOpAttrType type,
-                    _Outptr_ OrtOpAttr** op_attr);
+  OrtOpAttr* CreateOpAttr(_In_ const char* name,
```

Comment:
What's the reason for having this new version? All our APIs have pointers returned as out params so that we can return an appropriate status.

Reply:
Because for CustomOpApi, maybe we should make the API to be more CPP alike.
Which aligns the style with other functions in the struct like "GetTensorTypeAndShape" and "GetTensorData".

Reply:
Ah, this is in the cxx header. This is fine then.